src/transactions/script-composer/index.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { TransactionComposer, initSync, create_wasm } from "@wgb5445/aptos-intent-npm";
+import { AptosApiType } from "../../utils";
+import { AptosConfig } from "../../api/aptosConfig";
+import {
+  EntryFunctionArgumentTypes,
+  FunctionABI,
+  InputBatchedFunctionData,
+  SimpleEntryFunctionArgumentTypes,
+} from "../types";
+import { convertArgument, fetchMoveFunctionAbi, getFunctionParts, standardizeTypeTags } from "../transactionBuilder";
+import { TypeTag } from "../typeTag";
+import { CallArgument } from "../../types";
+
+(async () => {
+ initSync(await create_wasm());
+})();
+
+function convertCallArgument(
```

Comment:
lets move this file to live [here](https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/transactions/transactionBuilder/remoteAbi.ts#L196) as it is a wrapper for the underline `convertArgument` function.
Also, 
1. Please `export` it
2. Add a doc string
```suggestion
export function convertCallArgument(
```